protocols/kad/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
   See [PR 4934](https://github.com/libp2p/rust-libp2p/pull/4934).
 - Add periodic and automatic bootstrap.
   See [PR 4838](https://github.com/libp2p/rust-libp2p/pull/4838).
+  See [PR 5349](https://github.com/libp2p/rust-libp2p/pull/5349).
```

Comment:
Yeah of course. But do you think it is better ? Since we did not yet release `0.46.0`, I thought it would be weird if there is an entry "add feature" and then "fix feature".

What do you think ?

Reply:
Even though its not released, adding an entry under the [unreleased] version would be preferred as it is a different change.

Reply:
Ok. I am changing that :wink: